torch/csrc/jit/pybind_utils.h

```diff
@@ -235,6 +235,11 @@ inline IValue toIValue(
       }
       return userObj;
     }
+    case TypeKind::ProfiledTensorType:
```

Comment:
nit: same is true of DimensionedTensorType and CompleteTensorType but they are handled differently here. This needs to be consistent, probably by making ProfiledTensorType behave the same as DimensionedTensorType here.